src/core/bpf/socket_bind/meson.build

```diff
@@ -1,14 +1,48 @@
 # SPDX-License-Identifier: LGPL-2.1+
 
 if conf.get('BPF_FRAMEWORK') == 1
+        clang_flags = [
+                '-Wno-compare-distinct-pointer-types',
+                '-O2',
+                '-target',
+                'bpf',
+                '-g',
+                '-c',
+        ]
+
+        clang_arch_flag = '-D__@0@__'.format(host_machine.cpu_family())
+
+        socket_bind_bpf_o_unstripped = custom_target(
```

Comment:
src/core/bpf/socket_bind/meson.build:15:8: ERROR: Function does not take positional arguments.

Reply:
Hmm, I've been unable to reproduce that locally, maybe it's an issue with older meson versions?

Reply:
0.55 is still supported, so it needs to be fixed

Reply:
I think I managed to work around this error.